src/extensions/uptime.py

```diff
@@ -0,0 +1,15 @@
+import time as t; starttime = t.time()
+import arc
+import hikari
+import datetime as dt
+
+upt_plugin = arc.GatewayPlugin("Blockbot Uptime")
+
+@upt_plugin.include

```

Comment:
Code should be readable by design!

```suggestion
@upt_plugin.include
uptime_plugin.include
```
```